examples/v1beta1/pipelineruns/pipelinerun-task-execution-status.yaml

```diff
@@ -0,0 +1,56 @@
+kind: PipelineRun
+apiVersion: tekton.dev/v1beta1
+metadata:
+  generateName: pr-execution-status-
+spec:
+  serviceAccountName: 'default'
+  pipelineSpec:
+    tasks:
+    - name: task1 # successful task
+      params:
+      taskSpec:
+        steps:
+        - image: ubuntu
+          name: hello
+          script: |
+            echo "Hello World!"
+    - name: task2 # skipped task
+      when:
+        - input: "true"
+          operator: "notin"
+          values: [ "true" ]
+      taskSpec:
+        steps:
+          - image: ubuntu
+            name: success
+            script: |
+              exit 0
+    - name: task3 # failed task
+      taskSpec:
+        steps:
+          - image: ubuntu
+            name: fail
+            script: |
+              exit 1
+    finally:
+    - name: task4
+      params:
+        - name: pipelineRun-pipelineTask-task1
+          value: "$(context.pipelineRun.Tasks.task1)"
+        - name: pipelineRun-pipelineTask-task2
+          value: "$(context.pipelineRun.Tasks.task2)"
+        - name: pipelineRun-pipelineTask-task3
+          value: "$(context.pipelineRun.Tasks.task3)"
+      taskSpec:
+        params:
+          - name: pipelineRun-pipelineTask-task1
+          - name: pipelineRun-pipelineTask-task2
+          - name: pipelineRun-pipelineTask-task3
+        steps:
+          - image: ubuntu
+            name: print-failed-task-status
+            script: |
+              echo "Pipeline Task Status: Task1:  $(params.pipelineRun-pipelineTask-task1)"
```

Comment:
Can we make this example fail if the reported status isn't what we expect? That might catch a future bug that would otherwise go undetected.

Reply:
yup definitely, any pointer on how to add failed run under `/examples`? I think we added support for it at some point 🤔 will it have to live under `/test`?

Reply:
I couldn't find anywhere 😭 I will move this test under `/test` but would like to keep this example file as well. Do we run examples from [no-ci](https://github.com/tektoncd/pipeline/tree/master/examples/v1beta1/pipelineruns/no-ci), may be move there?